source/blog/2018-12-21-the-ember-times-issue-78.md

```diff
@@ -42,8 +42,21 @@ thoughts!
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [RFC: Glimmer Components ✨](https://github.com/emberjs/rfcs/pull/416)
+
+[@pzuraq](https://github.com/pzuraq) put up an RFC for supporting **Glimmer components** – which are simpler, more declarative and ergonomic approach to building components – into Ember’s public API. Glimmer components will be the new default app experience in [Ember Octane](https://github.com/emberjs/rfcs/blob/26c4d83fb66568e1087a05818fb39a307ebf8da8/text/0000-roadmap-2018.md#ember-octane).
```

Comment:
I think this might be

```suggestion
[@pzuraq](https://github.com/pzuraq) put up an RFC (Request for Comments) for supporting **Glimmer components** – which represent a simpler, more declarative and ergonomic approach to building components – into Ember’s public API. Glimmer components will be the new default app experience in [Ember Octane](https://github.com/emberjs/rfcs/blob/26c4d83fb66568e1087a05818fb39a307ebf8da8/text/0000-roadmap-2018.md#ember-octane).
```